build/images/base/Dockerfile

```diff
@@ -13,6 +13,27 @@
 # limitations under the License.
 
 ARG BUILD_TAG
+
+FROM ubuntu:22.04 AS bgp
+
+ARG BGP_VERSION
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends wget ca-certificates
+
+RUN set -eux; \
+    dpkgArch="$(dpkg --print-architecture)"; \
+    case "${dpkgArch##*-}" in \
+         amd64) pluginsArch='amd64' ;; \
+         armhf) pluginsArch='arm' ;; \
+         arm64) pluginsArch='arm64' ;; \
+         *) pluginsArch=''; echo >&2; echo >&2 "unsupported architecture '$dpkgArch'"; echo >&2 ; exit 1 ;; \
+         esac; \
+	 mkdir -p /opt/bgp; \
+         wget -q -O - https://github.com/osrg/gobgp/releases/download/v$BGP_VERSION/gobgp_${BGP_VERSION}_linux_${pluginsArch}.tar.gz | tar xz -C /opt/bgp; \
+         find /opt/bgp -type f ! -name 'gobgp' -delete
```

Comment:
not necessary IMO, for an intermediate image

Reply:
For intermediate image, we can skip this